src/core/keybindings.c

```diff
@@ -1553,13 +1553,15 @@ process_keyboard_move_grab (MetaDisplay *display,
     case XK_KP_Prior:
     case XK_Up:
     case XK_KP_Up:
+	 case XK_k:
```

Comment:
Much better, thanks - now you still need to adjust the indentation so that it matches the existing code

Reply:
strange seems even on my end even on my end. i'll see whats going on there.

Reply:
hmm must be something up with my vim settings seems even on my end but not in the changes